Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay.js';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import MathButtons from './components/ButtonComponents/MathButtons'
+
+const mathSymbols = [{
+    text: '÷',
+  },{
+    text: '×',
+  },{
+    text: '-',
+  },{
+    text: '+',
+  },{
+    text: '=',
+  },
+]
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className='container'>
+      <CalculatorDisplay />
+      <div className='work'>
+        <div className='work-buttons'>
+          {mathSymbols.map(symbol => {
```

Comment:
A unique key should be used for each element in the list.
https://reactjs.org/docs/lists-and-keys.html